test/widgets/action_sheet_test.dart

```diff
@@ -108,21 +108,26 @@ void main() {
   }
 
   group('topic action sheet', () {
+    final someChannel = eg.stream();
+    const someTopic = 'my topic';
+
     group('showTopicActionSheet', () {
-      final channel = eg.stream();
-      const topic = 'my topic';
       final message = eg.streamMessage(
-        stream: channel, topic: topic, sender: eg.otherUser);
+        stream: someChannel, topic: someTopic, sender: eg.otherUser);
 
       Future<void> prepare({
+        ZulipStream? channel,
```

Comment:
I think none of the callers of `prepare` ends up needing `channel` to be a specific value other than `someChannel`, so maybe we can get this removed?

Reply:
I think I slightly prefer leaving it in, in case later tests want to use it? It would be easy to remove, but harder to add back correctly (without forgetting e.g. to use `effectiveChannel` everywhere instead of just `channel`, which the analyzer wouldn't help with because they're both in scope).